examples/siro_sandbox/sandbox_app.py

```diff
@@ -790,15 +790,23 @@ def _parse_debug_third_person(args, framebuffer_size):
             ),
         )
 
+    # TextDrawer kwargs
+    display_font_size = 20
+    font_size_multiplier = (
+        gui_app_wrapper.framebuffer_size
+        * gui_app_wrapper.dpi_scaling
+        / gui_app_wrapper.window_size
```

Comment:
```suggestion
        mn.Vector2(gui_app_wrapper.framebuffer_size)
        * gui_app_wrapper.dpi_scaling
        / mn.Vector2(gui_app_wrapper.window_size)
```

Your code will work correctly only with an integer ratio of the framebuffer and window sizes, casting to a floating-point type makes it work with 1.5x scale and such too (I have a 1.6875x scale on my external display, for example).

Reply:
Thank you @mosra!